pom.xml

```diff
@@ -59,6 +60,27 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
```

Comment:
No need to enumerate all the log4j deps in the `dependencies` section.
Could you please update `dependencyManagement` section instead:

```xml
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>org.apache.logging.log4j</groupId>
                <artifactId>log4j-bom</artifactId>
                <version>2.16.0</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>

            <dependency>
                <!-- Import dependency management from Spring Boot -->
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-starter-parent</artifactId>
                <version>${spring.boot.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
```

Reply:
You are right. Now it is working. (maybe order of dependencies is important? this could be the reason why before it doesn't work before.
Anyway thank you for the feedback.